manifests/metering-config/azure-blob-storage.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: metering.openshift.io/v1
+kind: MeteringConfig
+metadata:
+  name: "operator-metering"
+spec:
+  storage:
+    type: "hive"
+    hive:
+      type: "azure"
+      azure:
+        container: "bucket1"
+        storageAccountName: "meteringazure"
+        secretAccessKey: "your-secret-here"
```

Comment:
Our default example should instruct the user to create a secret and reference it, instead of putting their credentials in the MeteringConfig